circle.yml

```diff
@@ -0,0 +1,36 @@
+machine:
```

Comment:
Yes, please.
I added a service but for some reason push didn't trigger a build. Maybe because there's no `circle.yml` in the master. I don't know.